src/lib/modules/console/index.ts

```diff
@@ -78,6 +78,16 @@ class Console {
         "swarm - " + __("instantiated swarm-api object configured to the current environment (available if swarm is enabled)"),
         "web3 - " + __("instantiated web3.js object configured to the current environment"),
         "EmbarkJS - " + __("EmbarkJS static functions for Storage, Messages, Names, etc."),
+        "token - " + __("Copies and prints the token for the cockpit"),
+        "debug - " + __("Debug the last transaction or the transaction specified by a hash"),
```

Comment:
shall you say:

"debug [transactionHash]" it will be more obvious a prams is expected